README.md

```diff
@@ -41,112 +43,120 @@ conctr <- Conctr(APP_ID, API_KEY, MODEL);
 
-##### Example
+#### Example
 
 ```squirrel
-const CUSTOM_DEVICE_ID="device-1";
+const CUSTOM_DEVICE_ID = "device-1";
 
 conctr.setDeviceId(CUSTOM_DEVICE_ID);
 ```
 
+
 ### sendData(*payload[, callback]*)
 
-The *sendData()* method sends a data payload to Conctr via the data ingeston endpoint. It is called by the data event listener when the device sends data using the device Conctr class. It can also be used directly to send data to Conctr via the agent alone.
+The *sendData()* method sends a data payload to Conctr via the data ingeston endpoint. 
 
-| Key | Data type | Required | Description |
-| ----| --------------- | --------- | ----------- |
-| *payload* | Table/Array of Tables | Yes | A table or array containing the data to be sent to Conctr. The keys of each table in the data should correspond to fields from the model and/or Conctr metadata fields and should be of the type specified in the model.|
-| *callback* | Function | No | Function to be called on response from Conctr. function should take two arguements, error and response. When no error occurred the first arguement will be null.|
+| Key | Data Type | Required | Description |
+| --- | --------- | -------- | ----------- |
+| *payload* | Table/Array | Yes | A table or array of tables containing the data to be sent to Conctr. The keys in the table should correspond to fields from the model and the keys should be of type specified in the model. |
+| *callback* | Function | No | Function to be called on response from Conctr. The function should take two arguements, *error* and *response*. When no error occurred, the first arguement will be null. |
 
-##### Example
+#### Example
 
 ```squirrel
-local curTempAndPressure = { "temperature" : 29, "pressure" : 1032};
+local currentTempAndPressure = { "temperature" : 29, "pressure" : 1032};
 
-conctr.sendData(curTempAndPressure, function(error, response) {
+conctr.sendData(currentTempAndPressure, function(error, response) {
     if (error) {
-        //handle error
+        // Handle error
     } else {
-        //data was successfully recieved by Conctr
+        server.log("Data was successfully recieved by Conctr");
     }
 }.bindenv(this));
 ```
 
 ## Device Class Usage
+**NOTE:** The device class is optional. It provides utility functions for interfacing with the agent class like automating the location sending process and provide queueing and error handling for sending data to conctr.
+### Constructor: Conctr(*[options]*)
+
+Instantiates the Conctr device class. It takes an optional table used to set the configuration of the class. *options* may contain any of the following keys:
+
+| Key | Data Type | Default Value | Description |
+| --- | --------- | ------------- | ----------- |
+| *options.locEnabled* | Boolean | `true` | When enabled, location data will be automatically included with the data payload. |
```

Comment:
inconsistent with Description ending in a full stop. I think it looks better without a full stop